packages/ng-form/README.md

```diff
@@ -0,0 +1,102 @@
+# ng-form
+
+An Angular library to ease the management of Angular Forms.
```

Comment:
The doc is not completed yet, it misses the description of Validators and Utilities. For know we keep the doc as it is as I need to move to another project